src/main/java/org/codehaus/plexus/archiver/tar/PlexusIoTarSnappyFileResourceCollection.java

```diff
@@ -1,7 +1,11 @@
 package org.codehaus.plexus.archiver.tar;
 
+import javax.inject.Named;
+
 import java.io.File;
 
+// TODO: Name is okay? Seems not
+@Named( "snappy.gz" )
```

Comment:
this look totally wrong. I'd expect `tar.snappy`

Reply:
All this comes from (under maintained) plexus XML file, there was even `par` vs `Par`
https://github.com/codehaus-plexus/plexus-archiver/blob/337214c2976564ff4255882bd76954c331a4e62e/src/main/resources/META-INF/plexus/components.xml#L183
https://github.com/codehaus-plexus/plexus-archiver/blob/337214c2976564ff4255882bd76954c331a4e62e/src/main/resources/META-INF/plexus/components.xml#L378

Reply:
https://github.com/codehaus-plexus/plexus-archiver/blob/337214c2976564ff4255882bd76954c331a4e62e/src/main/resources/META-INF/plexus/components.xml#L405

Reply:
:-D